Deleteusercomment

Greeny1248 opened the pull request:
Added more CSS styling (please bare with me while sorting the colour scheme of light and dark more, will sort tomorrow afternoon), added logged in user with the ability to delete comments which correspond with the content author   (will add username log in screen once complete).

Greeny1248 committed:
formatting

Greeny1248 committed:
git Merge branch 'main' of https://github.com/Greeny1248/NC-games into categoryfilter

Greeny1248 committed:
more CSS styling, added more use functionality and ability to delete if logged in user = comment author

MKD01 commented on src/components/comments.jsx:
```diff
@@ -63,7 +71,8 @@ export const Comments = () => {
               <br></br>
               {comment.votes} Vote
-              {comment.username === loggedInUsername ? (
+              {console.log(comment.author)}
```
extra console.log

Greeny1248 replied:
spotted these after pr thanks mkd!

MKD01 commented on src/components/comments.jsx:
```diff
@@ -2,13 +2,21 @@ import { useParams } from "react-router-dom";
+  
+  
+  const userValue = useContext(UserContext);
+  console.log(userValue.loggedInUsername.name)
```
extra console.log

Greeny1248 replied:
spotted these after pr thanks mkd!

MKD01 reviewed (approved):
Great stuff so far but one extra bit that can be added would be an error message if the post request fails.

Once this has been added im happy for you to merge 👍

Greeny1248 commented:
![image](https://user-images.githubusercontent.com/118350397/217858412-bbd427b5-539c-4bc0-8619-05a3f38e51ce.png)

![image](https://user-images.githubusercontent.com/118350397/217858331-b14f098c-5c55-4e60-a087-d322fa678b72.png)

Added this feature and removed the console logs.
